.github/workflows/test.yml

```
@@ -107,86 +107,86 @@ jobs:
-          if [ -s gosec.sarif ]; then
-            echo "✅ SARIF file has content"
+      - name: 📥 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 🐹 Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
+
+      - name: 🛡️ Run Gosec Security Scanner
+        uses: actions/setup-go@v4
```

actionlint (1.7.8)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)